qutebrowser/utils/jinja.py

```diff
@@ -47,8 +90,8 @@ def get_source(self, _env, template):
         path = os.path.join(self._subdir, template)
         try:
             source = utils.read_file(path)
-        except OSError:
-            raise jinja2.TemplateNotFound(template)
+        except OSError as e:
+            source = html_fallback + str(e) + html_fallback2;
```

Comment:
Please use `html.escape(str(e))` to have proper HTML escaping (with the `html` module in the stdlib)